client/signup/jetpack-connect/install-step.jsx

```diff
@@ -15,83 +16,100 @@ import JetpackExampleConnect from './example-components/jetpack-connect';
 
 const NEW_INSTRUCTIONS_JETPACK_VERSION = '4.2.0';
 
-export default React.createClass( {
-	displayName: 'JetpackInstallStep',
+class JetpackInstallStep extends Component {
 
```

Comment:
Usually our linter will complain about empty lines in the beginning of code blocks. For some reason that does not happen in classes. But let's remove that one regardless.